Terminal.Gui/ApplicationRunState.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Terminal.Gui {
+	/// <summary>
+	/// The execution state for a <see cref="Toplevel"/> view.
+	/// </summary>
+	public class ApplicationRunState : IDisposable {
```

Comment:
For terseness-sake, I think this should remain named RunState.

Reply:
Done